[2022-06-19 04:28:50,305] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: Job-pts.pts-Papple-iphone-xs-max-64gb-space-grau_h_6251 scheduled__2022-06-19T01:11:07.661205+00:00 [None]>
[2022-06-19 04:28:50,646] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: Job-pts.pts-Papple-iphone-xs-max-64gb-space-grau_h_6251 scheduled__2022-06-19T01:11:07.661205+00:00 [None]>
[2022-06-19 04:28:50,649] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-19 04:28:50,649] {taskinstance.py:1357} INFO - Starting attempt 1 of 6
[2022-06-19 04:28:50,649] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-19 04:28:53,183] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): pts-Papple-iphone-xs-max-64gb-space-grau_h_6251> on 2022-06-19 01:11:07.661205+00:00
[2022-06-19 04:28:53,825] {standard_task_runner.py:52} INFO - Started process 18990 to run task
[2022-06-19 04:28:54,120] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'Job-pts', 'pts-Papple-iphone-xs-max-64gb-space-grau_h_6251', 'scheduled__2022-06-19T01:11:07.661205+00:00', '--job-id', '536', '--raw', '--subdir', 'DAGS_FOLDER/docker_job/price_scale.py', '--cfg-path', '/tmp/tmps_yft231', '--error-file', '/tmp/tmp7c9twus0']
[2022-06-19 04:28:54,123] {standard_task_runner.py:80} INFO - Job 536: Subtask pts-Papple-iphone-xs-max-64gb-space-grau_h_6251
[2022-06-19 04:29:18,068] {warnings.py:110} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/configuration.py:525: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  option = self._get_environment_variables(deprecated_key, deprecated_section, key, section)

[2022-06-19 04:30:08,707] {standard_task_runner.py:97} ERROR - Failed to execute job 536 for task pts-Papple-iphone-xs-max-64gb-space-grau_h_6251 ((psycopg2.OperationalError) could not translate host name "postgres_source" to address: Temporary failure in name resolution

(Background on this error at: http://sqlalche.me/e/14/e3q8); 18990)
[2022-06-19 04:30:56,834] {local_task_job.py:156} INFO - Task exited with return code 1
[2022-06-19 04:30:57,173] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=Job-pts, task_id=pts-Papple-iphone-xs-max-64gb-space-grau_h_6251, execution_date=20220619T011107, start_date=20220619T042850, end_date=20220619T043057
[2022-06-19 04:30:57,371] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
